developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -51,6 +51,9 @@ npm install
 An error occurred while installing pg (1.4.1), and Bundler cannot continue.
 Make sure that `gem install pg -v '1.4.1' --source 'https://rubygems.org/'` succeeds before bundling.
 ...
+* In case you get this error:
+`extconf.rb:37:in `<main>': libvirt library not found in default locations (RuntimeError)`. Try to use this command: `export PKG_CONFIG_PATH=/usr/lib64/pkgconfig`.
```

Comment:
When did you get this error, when running `bundle install`? Which `gem` caused the issue, `pg`?

Reply:
I think it was `ruby-libvirt`